application/gui/properties_window.py

```diff
@@ -193,6 +197,8 @@ def _rename_item(self, widget=None, data=None):
 
 	def _create_monitor(self):
 		"""Create item monitor"""
+		if hasattr(self, '_monitor') and self._monitor is not None and not self._monitor.is_cancelled():
+			self._monitor.cancel()
```

Comment:
As I didn't actually see any bad happen (I guess that would take a few
monitors to be created if they don't get released properly) - do you want
me to to remove the two lines for the time being?

On Mon, May 2, 2016 at 11:48 AM, Mladen Mijatov <notifications@github.com>
wrote:

> In application/gui/properties_window.py
> <https://github.com/MeanEYE/Sunflower/pull/230#discussion_r61725247>:
>
> > @@ -193,6 +197,8 @@ def _rename_item(self, widget=None, data=None):
> >
> >  	def _create_monitor(self):
> >  		"""Create item monitor"""
> > +		if hasattr(self, '_monitor') and self._monitor is not None and not self._monitor.is_cancelled():
> > +			self._monitor.cancel()
>
> I am noticing that trend about pull requests. Can't say am sorry for that
> to be the case as I should be able to join in on development soon. As for
> this change, am not sure it's a good idea to _monitor like that. I'll
> have to look deeper into that so I'll wait a bit with merge.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/MeanEYE/Sunflower/pull/230/files/dc4966501ec1bcc599007680f92fcff3e7067830#r61725247>
>